dropgz/pkg/embed/payload.go

```diff
@@ -86,6 +86,19 @@ func deploy(src, dest string) error {
 		return err
 	}
 	defer rc.Close()
+
+	if _, err := os.Stat(dest); err == nil || !os.IsNotExist(err) {
+		old_file_dest := dest + ".old"
+		os.RemoveAll(old_file_dest)
```

Comment:
what happens if this remove fails? ignoring the returned error is 👎🏼, if we really don't care we should `_ = os.RemoveAll(old_file_dest)` it to positively indicate that it doesn't matter.
but if we don't care if this errors...do we even need it?

Reply:
I think it is better to add an error handler than to silently ignore it. So will be adding it in the next update.

Reply:
can you explain why or why not we should handle this error?

Reply:
We should be handling the error in this case, `Removeall` will return nil if the path does not exists which is good for us for the first time. But there can be other Patherror returned while removing the dest. If we ignore it, then the next part of the code that renames the files will be errored out as well because the `old` file was not removed in the first place.

Reply:
why dont we use `os.Remove` since we are removing single file?

Reply:
For the first the file won't be present, so `os.Remove` will error out.